target/linux/layerscape/armv8_64b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=aarch64
```

Comment:
If the SoC supported by this target all have an cortex A53 CPU, please add:
CPU_TYPE:=cortex-a53